src/app/components/diary-card/diary-card.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-diary-card',
+  templateUrl: './diary-card.component.html',
+  styleUrls: ['./diary-card.component.scss']
+})
+export class DiaryCardComponent implements OnInit {
+
+  constructor() { }
+  title:string = "title";
```

Comment:
Make sure to define the access modifier 
`public title: string = "....."`

Fix this in everywhere.

Reply:
fixed